### DISCUSSION TITLE
Requirements: Node version

### DISCUSSION THREAD

**Discussion by abicorios** [ChatGPTNextWeb/NextChat#2095] [Q&A]
Can we add to the `README.MD` the required Node version?

I see in the file `package.json`, the `@types/node` were `^18.14.6`, then it was updated to `^20.3.1`. So I thought, I need the Node `18.14.6`, and now I think, maybe I need to switch to the Node `20.3.1`.

But, in the `Dockerfile` I see `FROM node:18-alpine`, so I think I need the Node `18`.

But, in the `.github\workflows\app.yml` file I see `node-version: 16`, so maybe Node `16` is okay too.

Someone can think, it is not important. But, some error messages and library version resolutions are depend on Node version. And error messages are depend on library versions. We can see different error messages, because different Node versions. And, if we will select one good version, we will see same, and it will be simpler to fix.

I used Node `18.14.2` for Windows `10`, and Node `18.16.0` for Debian `11`, it was working okay.

**Comment by Yidadaa** [comment]
Updated: https://github.com/Yidadaa/ChatGPT-Next-Web/blob/main/README.md#requirements
